packages/embark/src/lib/utils/solidity/remapImports.ts

```diff
@@ -101,8 +96,8 @@ const rescursivelyFindRemapImports = async (file: File, filesProcessed: string[]
 
   for (const importPath of imports) {
     const newFile = buildNewFile(file, importPath);
-    file.importRemappings.push({prefix: importPath, target: newFile.path});
-    remapImports.push({path: file.path, searchValue: importPath, replaceValue: newFile.path});
+    file.importRemappings.push({ prefix: importPath, target: newFile.path });
+    remapImports.push({ path: file.path, remapping: { prefix: importPath, target: newFile.path } });
```

Comment:
the import remappings can be saved in a constats, so you don't have to build it 2 times

Reply:
Updated, thanks!